modules/trusted-firmware-m/CMakeLists.txt

```diff
@@ -249,13 +249,27 @@ if (CONFIG_NRF_APPROTECT_LOCK)
   )
 endif()
 
+if (CONFIG_NRF_APPROTECT_USER_HANDLING)
+  set_property(TARGET zephyr_property_target
+  APPEND PROPERTY TFM_CMAKE_OPTIONS
+    -DCONFIG_NRF_APPROTECT_USER_HANDLING=ON
+  )
```

Comment:
```suggestion
  set_property(TARGET zephyr_property_target
               APPEND PROPERTY TFM_CMAKE_OPTIONS
               -DCONFIG_NRF_APPROTECT_USER_HANDLING=ON
  )
```
should line up

Reply:
n.b. line length should be <= 100 characters so you can put more of these onto previous lines

Reply:
From what I see, this indentation is new in the code

I think it is more common to follow the CMake 2 character indentation

Reply:
@nordicjm: CMake 2 character indentation seems to be the default in TF-M codes. I would like to revert my whitespace changes related to this.

Reply:
This isn't code in tfm, this is code in ncs

Reply:
Although the code is located in ncs, it is compiled in TF-M. The TF-M is split between the module and the code located in here and they are compiled together to achieve end result. When you are working with this code, you are usually also working with TF-M module code, so it would be beneficial to have same indentation used.